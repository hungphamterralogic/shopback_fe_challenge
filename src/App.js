import React from 'react';
import CategoriesAndStores from './scenes/categoriesAndStores';

const App = () => {
  return <CategoriesAndStores />;
};

export default App;
