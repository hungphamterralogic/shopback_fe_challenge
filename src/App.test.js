import React from 'react';
// import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import App from './App';
import configureStore from './store';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(
//     <Provider store={configureStore()}>
//       <App />
//     </Provider>,
//     div
//   );
//   ReactDOM.unmountComponentAtNode(div);
// });

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={configureStore()}>
        <App />
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
