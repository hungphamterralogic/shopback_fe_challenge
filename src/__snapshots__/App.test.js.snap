// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="CategoriesAndStores-wrapper-0-1-1"
  >
    <div
      className="TabBar-tabBar-0-1-2"
    />
    <div
      className="StoreList-storeList-0-1-5"
    >
      <div
        className="ShowMore-showMoreDesktop-0-1-6"
      >
        <div
          className="ShowMore-textBoxDesktop-0-1-8"
        >
          view all popular stores
        </div>
      </div>
    </div>
  </div>
</div>
`;
